MapboxNavigation/NavigationViewController.swift

```diff
@@ -34,6 +34,9 @@ public protocol NavigationViewControllerDelegate {
     @objc(navigationViewController:shouldRerouteFromLocation:)
     optional func navigationViewController(_ navigationViewController: NavigationViewController, shouldRerouteFrom location: CLLocation) -> Bool
     
+    @objc(navigationViewController:shouldIncrementLegWhenArrivingAtWaypoints:)
```

Comment:
This public method needs a documentation comment. (Same goes for the corresponding method on RouteController.)